spec/data/acceptance/css/u4p1.css

```diff
@@ -3,36 +3,36 @@ body{
     background-size:cover;
     color:silver;
     }
-    #banner{
-        position:relative;
-        top:20px;
-        left:250px;
-        }
-        #tagline{
-            position:relative;
-            top:600px;
-            left:250px;
-            }
-            img{
-            position:relative;
-            top:10px;
-            left:10px;
-            height:100px;
-            width:500px;
-            }
-            #mission{
-            position:relative;
-            top:300px;
-            left:100px;
-            }
-            #about{
-            position:relative;
-            top:100px;
-            left:300px;
-            }
-            #topics{
-            position:relative;
-            top:300px;
-            left:400px;
-            }
-    
\ No newline at end of file
+#banner{
+    position:relative;
+    top:20px;
+    left:250px;
+}
+#tagline{
+    position:relative;
+    top:600px;
+    left:250px;
+}
+img{
+    position:relative;
+    top:10px;
+    left:10px;
+    height:100px;
+    width:500px;
+    filter: contrast(175%) brightness(3%);
+}
+#mission{
+    position:relative;
+    top:300px;
+    left:100px;
+}
+#about{
+    position:relative;
+    top:100px;
+    left:300px;
+}
+#topics{
+    position:relative;
+    top:300px;
+    left:400px;
+}
```

Comment:
I only changed spacing. Part of that css file was oddly tabbed out.
I also added the `filter` property on line 22.

Sure I can also add it in the CSS spec.